sklearn/neighbors/graph.py

```diff
@@ -44,6 +65,11 @@ def kneighbors_graph(X, n_neighbors, mode='connectivity', metric='minkowski',
         The default distance is 'euclidean' ('minkowski' metric with the p
         param equal to 2.)
 
+    include_self: bool, default None
```

Comment:
The default option is not True. In mode distance, the original behavior was include_self = False